app/create_file.py

```diff
@@ -1 +1,56 @@
-# write your code here
+from datetime import datetime
+from sys import argv
+from os import mkdir, chdir
+
+filename = ""
+directories = []
+
+if "-f" in argv:
+    index = 0
+
+    for _ in range(len(argv) - 1):
+        index += 1
+        if argv[_: _ + 1] == ["-f"]:
+            break
+
+    filename = argv[index]
```

Comment:
it'd be the wrong filename.
firstly, you need to add to index 3, because `argv[index]` == '-', `argv[index+1]` == 'f', `argv[index+2]` == ' '
also you need to make slicing to the first space using `find()` method
so it will look like:
`file_end_index = argv.find(' ', file_start_index)`
`filename = argv[file_start_index+3: file_end_index]`